### PR TITLE
compal

### PR DESCRIPTION
**Pull request by lymantseng**
### Thank you for contributing to the Azure IoT Plug and Play Models repository

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

The repository pipeline will ensure minimum validation of incoming DTDL models.  

- PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).
- The same validation can be run locally via the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet#device-model-repository-client) CLI.

## Requested info template for model(s) submission

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve Plug and Play.

:star2: Please replace the markdown comment examples with your own values.

### CODE REVIEW

File: dtmi/compal/pixsee/2.indicatorstate.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (rido-min) [inline dtmi/compal/pixsee/2.indicatorstate.json:1]: @lymantseng  thanks for the updates. 

Please fix the file names, they should be `dtmi/compal/pixsee/indicatorstate-1.json`, and please dont forget to sign the CLSA !!

**Review by rido-min — CHANGES_REQUESTED** [review]
fix file names

File: dtmi/compal/pixsee/2.indicatorstate.json (hunk 1 above)
Comment (lymantseng) [reply dtmi/compal/pixsee/2.indicatorstate.json:1]: Already sing the CLSA and rename the file name

Comment (rido-min) [reply dtmi/compal/pixsee/2.indicatorstate.json:1]: I'd suggest you to use the `dmr-client` CLI tool (requires .NET SDK installed) to import and validate the files prior submissions. The filenames are not following the convention. You can see the failing files in the PR checks.

Comment (lymantseng) [reply dtmi/compal/pixsee/2.indicatorstate.json:1]: Already fix the Contents and use dmr-client CLI tool check

Comment (rido-min) [reply dtmi/compal/pixsee/2.indicatorstate.json:1]: `dtmi/compal/pixsee/1.pixsee.json` should be `dtmi/compal/pixsee/pixsee-1.json` 
Can you please rename those files?

Comment (lymantseng) [reply dtmi/compal/pixsee/2.indicatorstate.json:1]: 圖片
Hi, Could you tell me where should changed?

**Comment by lymantseng** [comment]
Hi , Could you assist us to close the pull request, and then I do the request again?

**Pull request closed without merging**